slim_model.py

```diff
@@ -11,13 +11,15 @@
 from mesh_transformer.transformer_shard import CausalTransformer
 from smart_open import open
 
-from mesh_transformer.util import clip_by_global_norm, to_bf16
+from mesh_transformer.util import clip_by_global_norm, to_bf16, to_f16
```

Comment:
I think you forgot to include your to_f16 function

Reply:
Oh whoops, I thought it was already in the repo... pushed